packages/web-config-server/src/apiV1/dataBuilders/generic/percentage/percentInGroupByFacility.js

```diff
@@ -6,124 +6,105 @@ import {
   getPacificFacilityStatuses,
   limitRange,
 } from '/apiV1/utils';
+import { DataBuilder } from '/apiV1/dataBuilders/DataBuilder';
 
 // Medicines available by Clinic
 // Medicines available by Country
-export const percentInGroupByFacility = async (
-  { models, dataBuilderConfig, query, entity, fetchHierarchyId },
-  aggregator,
-) => {
-  const { dataElementCodes, dataServices, countries, range } = dataBuilderConfig;
-  const { results, period } = await aggregator.fetchAnalytics(
-    dataElementCodes,
-    { dataServices },
-    query,
-  );
+class PercentInGroupByFacilityBuilder extends DataBuilder {
```

Comment:
I won't take the time to thoroughly review that, let's add it in the regression testing notes!

Reply:
It's only used in two reports, and I've regression tested them both against dev already - but I will add an extra note so it's not just my overconfident eyes ;-)